code/frameworks/react-vite/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "rootDir": "./src",
     "types": ["node"],
     "resolveJsonModule": true,
-    "strict": true
+    "skipLibCheck": true
```

Comment:
what was this change necessary for?

Reply:
Good question. This PR is mostly changes from #22324 and I guess this is one of those.

Reply:
This was done, because the TS compiler options here are different than the settings in the react-docgen repo. This lead to one of the CI jobs to fail, because the it was validating react-docgen types with the different options (mostly strict stuff). I fixed a lot of it and enabled some more strict options in react-docgen, so maybe it works now.